tox.ini

```diff
@@ -20,7 +26,7 @@ deps =
     coverage
 
 [testenv:flake8]
-basepython = python3.6
+basepython = python3.8
```

Comment:
Lately I've gotten into the habit of using `basepython = python3` for these ancillary tox envs, just so I don't have to keep bumping them.

Reply:
good idea, changed it.